web/src/components/LogosList/LogosList.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from "next/image";
+import { FC } from "react";
+
+import { BadgingFields } from "@/feeds/publications/types";
+import { slugify } from "@/utils/url";
+
+import styles from "./LogosList.module.scss";
+
+export type LogosListProps = {
+	logosList: BadgingFields[];
+	productId: string;
+	logoType: string;
+};
+
+export const LogosList: FC<LogosListProps> = ({
+	logosList,
+	productId,
+	logoType,
+}) => {
+	return logosList && logosList.length > 0 ? (
+		<div className={styles.div}>
+			{logosList.map((item, index) => (
```

Comment:
Could we give `styles.div`, `styles.span` and `styles.img` more semantic names? Maybe `styles.logoContainer`, `styles.logoHolder` and `styles.logo`?